scipy/sparse/linalg/_propack/setup.py

```diff
@@ -17,15 +17,32 @@ def check_propack_submodule():
 def configuration(parent_package='', top_path=None):
     from numpy.distutils.system_info import get_info, NotFoundError
     from numpy.distutils.misc_util import Configuration
-    config = Configuration('_propack', parent_package, top_path)
-    lapack_opt = get_info('lapack_opt')
+    from scipy._build_utils import (gfortran_legacy_flag_hook,
+                                    blas_ilp64_pre_build_hook,
+                                    get_f2py_int64_options,
+                                    get_g77_abi_wrappers,
+                                    uses_blas64,
+                                    needs_g77_abi_wrapper)
+
+    if uses_blas64():
+        lapack_opt = get_info('lapack_ilp64_opt', 2)
```

Comment:
@pv What would it look like to add support for ilp64 here?  Looking through a lot of the other ilp64 PRs, it appears that function signatures have been changed to use typedef'd integer types.  Would a lot of PROPACK sources need to be changed for that to happen here?  (not that it wouldn't be worth it or doable, I'm just not familiar with this and would need help formulating a strategy)